src/tools.ts

```diff
@@ -150,28 +135,36 @@ export function convertMcpToolToLangchainTool(
  */
 export async function loadMcpTools(
   client: Client,
-  responseFormat: ResponseFormat = 'text'
+  responseFormat: ResponseFormat = 'content',
```

Comment:
could we make this `content_and_artifact`

Reply:
FWIW, I'm also not even sure why this is an argument to this method. The consumer of this lib shouldn't be calling this directly, and if MCP tools can produce artifacts, I'd expect us to determine that from the tool metadata reported by the server.

Reply:
or, rather, i think we _should_ return a 2-item list always